src/client/firefox.js

```diff
@@ -7,12 +7,20 @@
 import { setupCommands, clientCommands } from "./firefox/commands";
 import { setupEvents, clientEvents } from "./firefox/events";
 import { features } from "../utils/prefs";
+import type { Grip } from "debugger-html";
+let DebuggerClient;
+
+function createObjectClient(grip: Grip) {
+  return DebuggerClient.createObjectClient(grip);
```

Comment:
Mochitest is failing here because the assignment in the `onConnect` below hadn't happened yet. Should you put a guard in to check if the client isn't undefined?

Reply:
I think it was failing because of https://bugzilla.mozilla.org/show_bug.cgi?id=1430799, which now landed in mozilla-central.
Locally, all the mochitest pass with `--verify` :)

Reply:
Ah good to know. I saw the failure complaining about undefined and figured it was some async funkiness. Guess we just need to update the CI image?

Reply:
i'm not familiar with this, but if this mean the CI image needs to be updated to pick up mozilla-central changes, then yes :)

Reply:
@nchevobbe you can update the `.travis.yml` w/ the new mc scha